content/en/docs/concepts/project-maturity.md

```diff
@@ -12,6 +12,13 @@ in both production and non-production clusters.
 
 The cert-manager API is currently at `v1` and is stable.
 
+## Supported Versions
+
+cert-manager supports versions for the current and last minor release.  This
+means that if the latest release is `v1.2`, cert-manager supports the versions
+`v1.2` and `v1.1`. Support means that bug fixes will be backported to supported
+minor versions and patch releases will be made.
```

Comment:
But why support the last minor release? 
Maybe add a note about the rationale, perhaps in terms of providing support for releases in the last X months.
But does this mean that upon releasing 1.3, we will no longer support 1.1 and no longer support people trying to run CM on K8S < 1.16?

OR

Are we making this policy change in order to continue to support people on older Kubernetes, which are now only supported by cert-manager 1.1.x, in which case, the docs should say something about that.

In https://github.com/jetstack/cert-manager/issues/3413 @meyskens proposed calling it 1.1 LTS, but if that is our policy it would be good to add a paragraph explaining how long we will support that LTS branch.

My2c: I personally don't think we should change the policy. I like the current policy of only supporting the current released version.